javascript-bank/account.js

```diff
@@ -0,0 +1,43 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if ((amount <= 0) || (typeof amount !== 'number') || (isNaN(amount)) || (!(Number.isInteger(amount)))) {
+    return false;
+  } else if (amount > 0) {
+    var newTransaction = new Transaction('deposit', amount);
+    this.transactions.push(newTransaction);
+    return true;
+  }
+};
+
+Account.prototype.withdraw = function (amount) {
+  if ((amount <= 0) || (typeof amount !== 'number') || (isNaN(amount)) || (!(Number.isInteger(amount)))) {
+    return false;
+  } else if (amount > 0) {
```

Comment:
Same thing here, no `else if` block necessary, this can just be done below the above conditional.